docs/scraping/introduction.md

```diff
@@ -0,0 +1,294 @@
+
+What's the connection to Pseudo URLs? Well, first, all the URLs found in the elements that match the link selector are collected. Then, Pseudo URLs are used to filter those URLs an enqueue only the ones that match the Pseudo URL structure. Simple.
+
+To scrape all the actors in store, we should use the Link selector to further filter the links that our Pseudo URL matches. For example, we're not interested in the following URL:
+
+    https://apify.com/docs/actor
+
+Even though it matches our Pseudo URL, it's not a link to an actor, but a link to documenation. To prevent links like those from being visited, we should specify a Link selector that filters them out. For now, let us just tell you that the Link selector you're looking for is:
+
+    div.item > a
+
+Save it as your Link selector. If you're wondering how we figured this out, just follow along with the tutorial. By the time we finish, you'll know why we used this selector too.
+
+### [](#test-run)Test run
+
+We've added some configuration, so it's time to test it. Just run the task, keeping the **Max pages per run** set to `10` and **Page function** the same. You should see in the log that the scraper first visits the Start URL and then several of the actor details, matching the Pseudo URL.
+
+### [](#the--code-pagefunction--code-)The `pageFunction`
+
+The Page function is a JavaScript function that gets executed for each page the scraper visits. To figure out how to create the `pageFunction`, you must first inspect the page's structure to get an idea of its inner workings. The best tools for that are Developer Tools in browsers, DevTools.
+
+#### [](#using-devtools)Using DevTools
+
+Open the [store page](https://apify.com/store) in the Chrome browser (or use any other browser, just note that the DevTools may differ slightly) and open the DevTools, either by right-clicking on the page and selecting `Inspect` or by pressing `F12`.
+
+The DevTools window will pop up, and display a lot of, perhaps unfamiliar, information. Don't worry about that too much and just open the Elements tab (the one with the page's HTML). The Elements tab allows you to browse the structure of the page and search within it using the search tool. You can open the search tool by pressing `CTRL+F` or `CMD+F`. Try typing `<title>` into the search bar.
+
+You'll see that the Element tab jumps to the first `<title>` element of the current page and that the title is `Store`. It's always good practice to do your research using the DevTools before writing the `pageFunction` and running your task.
+
+![devtools](https://apifyusercontent.com/b6ba4c89bd65705450f1fee33c37198186b175fc/68747470733a2f2f7261772e67697468756275736572636f6e74656e742e636f6d2f6170696679746563682f6163746f722d736372617065722f6d61737465722f646f63732f6275696c642f2e2e2f696d672f7573696e672d646576746f6f6c732e6a7067 "Finding title element in DevTools.")
```

Comment:
`apifyusercontent.com` links should be auto-generated from normal links, not embedded directly - the URLs can change!

Reply:
Thanks, fixed.